core/src/main/java/com/scalar/db/storage/jdbc/RdbEngineMysql.java

```diff
@@ -330,4 +334,17 @@ public String getEscape(LikeExpression likeExpression) {
     String escape = likeExpression.getEscape();
     return escape.isEmpty() ? "\\" : escape;
   }
+
+  @Override
+  public String tryAddIfNotExistsToCreateIndexSql(String createIndexSql) {
+    return createIndexSql;
+  }
+
+  @Override
+  public boolean isDuplicateIndexError(SQLException e) {
```

Comment:
There are some `isXxxxx()` methods above. How about moving this there?

Reply:
Yes, I moved this method with the other similar methods for all the `RdbEngineXXX.java`. Thank you.
https://github.com/scalar-labs/scalardb/pull/1125/commits/feb9fbc07f8a5933166f06d4309948ea9102ced2